isis/tests/FunctionalTestsMsi2isis.cpp

```diff
@@ -0,0 +1,469 @@
+#include <QTemporaryDir>
+
+#include "msi2isis.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Histogram.h"
+
+#include "gtest/gtest.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/msi2isis.xml").expanded();
+
+TEST(Msi2isis, Msi2isisTestGblIngestDefault) {
+  Pvl appLog;
+  QTemporaryDir prefix;
+  QString cubeFileName = prefix.path() + "/m0157063318f4_2p_iof_gbl.cub";
+  QVector<QString> args = {"from=data/near/msi2isis/m0157063318f4_2p_iof_gbl.lbl",
+                            "to=" + cubeFileName };
+
+  UserInterface options(APP_XML, args);
+  try {
+    std::cerr << "Running msi2isis..." << std::endl;
+    msi2isis(options, &appLog);
+    std::cerr << "Success!" << std::endl;
+
+  }
+  catch (IException &e) {
+    FAIL() << "Unable to ingest NEAR/MSI GBL file: " << e.toString().toStdString().c_str() << std::endl;
```

Comment:
No need to add `c_str`, gtest can take standard strings.

Reply:
Done.